web/package.json

```diff
@@ -100,13 +98,13 @@
     "@babel/plugin-transform-runtime": "^7.3.4",
     "@babel/preset-env": "^7.0.0",
     "@babel/preset-react": "^7.0.0",
-    "@storybook/react": "^3.4.11",
+    "@storybook/react": "^4.1.13",
```

Comment:
5.0.0 dropped today (what a coinkydink!).  I tried to go ahead and update to that, but it had some weird issues with core.js dependency version mismatches on _something_ downstream.  People have had luck getting it to work by adding a direct dependency on core.js, but...  I dunno, I opted not to use an hours-old version that didn't work out of the box.  That seems like what patch versions are for.  😛